requirements.txt

```diff
@@ -38,5 +38,6 @@ https://github.com/Harvard-University-iCommons/django-canvas-oauth/archive/v1.1.
 
 drf-api-tracking==1.8.4 # API tracking for DRF
 
-django-q2==1.9.0
+django-q2==1.8.0
 watchfiles==1.1.1
+openai==2.11.0
```

Comment:
v2.12.0 was released today

Reply:
his was updated as per last week and yes we do a new one today :). 
@jonespm do you want me to upgrade to latest version?

Reply:
Yes, I am going to